app/api/utility.py

```diff
@@ -414,43 +410,25 @@ def load_json(path: Path) -> dict:
         return json.load(f)
 
 
-def create_snomed_assessment_lookup(output_path: Path):
+def create_snomed_terms_lookup(vocab_path: Path, lookup_path: Path):
```

Comment:
1. not sure these parameter names are more intuitive. 
2. the function could use a more descriptive name as well

How about
```suggestion
def strip_snomed_terms_prefix(input_terms_path: Path, output_terms_path: Path):
```